tools/dumpobj.ml

```diff
@@ -169,7 +170,7 @@ let print_setglobal_name ic =
     if n >= Array.length !globals || n < 0
     then print_string "<global table overflow>"
     else match !globals.(n) with
-           Global id -> print_string(Ident.name id)
+         | Glob glob -> print_string (Symtable.Global.string_of_global glob)
          | _ -> print_string "???"
```

Comment:
I would make this pattern exhaustive

Reply:
The pattern could be made non-fragile, but that's seem quite orthogonal to this PR.